src/main/java/org/jenkinsci/test/acceptance/po/CapybaraPortingLayerImpl.java

```diff
@@ -95,23 +93,7 @@ protected final WebDriver visit(URL url) {
     @Override
     public void clickButton(String text) {
         WebElement e = find(by.button(text));
-        /*
-         * YUI sticky buttons present some problems when scroll to them, also if you have a sticky button
-         * you should not need to scroll to use them
-         */
-        boolean isStickyButton = false;
-        WebElement stickyContainer = getElement(by.id("bottom-sticker"));
-        if (stickyContainer != null) {
-            JavascriptExecutor je = (JavascriptExecutor)driver;
-            isStickyButton = (boolean)je.executeScript("return arguments[0].contains(arguments[1])", stickyContainer, e);
-        }
-        if (isStickyButton) {
-            Actions builder = new Actions(driver);
-            builder.moveToElement(e).click(e);
-            builder.perform();
-        } else {
-            e.click();
-        }
+        e.click();
```

Comment:
Love this :)

Reply:
Not sure why it was done since it is not my change, but I guess he had the reason, so I let it this way. But it probably caused the issue fixed in this commit https://github.com/jenkinsci/acceptance-test-harness/pull/457/commits/b45a4251583f5be64a1a8ac5d0754aef6acd3638

I will try to return it and disable my fix if it will be work old way without causing any other issue. Before it was hard to judge because there failed more then half tests. Now it should be visible, if it does not break anything, it should be better this way, thank you for point it.

Reply:
I tried it and it seems there is a problem with old implementation (which is the reason why it was removed), all tests fails with:

`The element reference of <button id="yui-gen7-button" type="button"> is stale; either the element is no longer attached to the DOM, it is not in the current frame context, or the document has been refreshed`

I look at it more to decide if there is an easy way how to fix it there (we can always let there my fix in Scroller). Or you have any suggestions?

Reply:
Oh sorry, my bad, I did mistake in returning it to the previous state. Now the tests are running.

Reply:
After returning to the previous state (with sticky buttons detection), there are a lot of more failure, usually with visiting page after saving configuration. But it does not seems as the failure of clicking on save button, but rather that something unexpected happens after clicking. So I am not sure if it is connected with it, but that is the only change. 

@rtyler Do you know why this was done?